integration-tests/hooks.ts

```diff
@@ -69,7 +69,7 @@ export function useCurrentBlockTimestampMultichain(
 
 export function useMaxTokenBalance(chainId: ChainId, blockNumber: number | undefined): string | undefined {
   const { contracts, accounts } = useMemo(
-    () => ({ contracts: [USDC_ADDRESS, USDT_ADDRESS], accounts: [NULL_ADDRESS] }),
+    () => ({ contracts: [, USDC_ADDRESS, USDT_ADDRESS, DAI_ADDRESS], accounts: [NULL_ADDRESS] }),
```

Comment:
Could you maybe add a comment above this line to specify why we're adding the empty element to this test? Could even link to this PR.